docker-compose.yml

```diff
@@ -7,6 +7,8 @@ services:
       PORT: 3000
       POSTGRES_HOST: "${POSTGRES_HOST:-db}"
       POSTGRES_PORT: "${POSTGRES_PORT:-5432}"
+      SIGNAL_REST_CLI_ENDPOINT: "http://signal:8080"
+      SIGNAL_REST_CLI_ATTACHMENT_PATH: "signal-cli-config/attachments/"
```

Comment:
It’s a little confusing that this is named `SIGNAL_REST_CLI` while the project is called `signal-cli-rest-api` (inverse order of cli and rest).